web/packages/teleterm/src/ui/DocumentGateway/useDocumentGateway.ts

```diff
@@ -124,6 +122,8 @@ export default function useGateway(doc: types.DocumentGateway) {
     connected,
     reconnect: createGateway,
     connectAttempt,
+    // TODO(ravicious): Show disconnectAttempt errors in UI.
+    disconnectAttempt,
```

Comment:
Exporting it here only because otherwise I wouldn't be able to wait for `disconnect` to finish in the test. But we should show these errors in the UI as well in case `disconnect` fails for any reason.